core/commands/active.go

```diff
@@ -25,7 +29,7 @@ Lists running and recently run commands.
 		res.SetOutput(req.InvocContext().ReqLog.Report())
 	},
 	Options: []cmdkit.Option{
-		cmdkit.BoolOption("verbose", "v", "Print extra information."),
+		cmdkit.BoolOption("verbose", verboseOptionName, "Print extra information."),
```

Comment:
We might want to switch this to constantize (and later use in req.Option()) the long name for this command so that it is consistent with all the others.